gorilla-minitwit/cmd/minitwit/main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"fmt"
+	"gorilla-minitwit/internal/config"
+	"gorilla-minitwit/internal/db"
+	"gorilla-minitwit/internal/handlers"
+	"gorilla-minitwit/internal/helpers"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/gorilla/mux"
+	"github.com/joho/godotenv"
+	_ "github.com/lib/pq"
+)
+
+var err error
+
+func main() {
+
+	/*---------------------
+	 * Load env vars
+	 *----------------------*/
+	dsn := LoadEnvVars()
+
+	/*---------------------
+	 * Connect to DB
+	 *----------------------*/
+	config.DB, err = db.ConnectDB(dsn) // Ensure this matches the function name in the db package
+	if err != nil {
+		log.Fatalf("Failed to connect to database: %v", err)
+	}
+
+	/*---------------------
+	* Setup routing
+	*----------------------*/
+	funcMap := SetupRouting()
+
+	config.Tpl, err = template.New("timeline.html").Funcs(funcMap).ParseGlob("../../templates/*.html")
+	if err != nil {
+		log.Fatalf("Error parsing template: %v", err)
+	}
+
+	/*---------------------
+	* Setup route-handlers
+	*----------------------*/
+	r := mux.NewRouter()
+	SetRouteHandlers(r)
+
+	fmt.Println("Listening on port 5000...")
+	err = http.ListenAndServe(":5000", r)
+	if err != nil {
+		log.Fatalf("Failed to start server: %v", err)
+	}
+}
+
+func SetupRouting() template.FuncMap {
```

Comment:
I think would be nice to have a folder routes in the internal folder with a routes.go including these functions.